<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mvd::ImportImagesDialog</class>
 <widget class="QDialog" name="mvd::ImportImagesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>275</width>
    <height>134</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configure multi-resolution pyramid</string>
  </property>
  <property name="locale">
   <locale language="C" country="AnyCountry"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="0" column="0">
    <widget class="QTreeView" name="filenamesTreeView"/>
   </item>
   <item row="1" column="0">
    <widget class="mvd::MultiResolutionPyramidWidget" name="pyramidWidget" native="true"/>
   </item>
   <item row="2" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ignore|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>mvd::MultiResolutionPyramidWidget</class>
   <extends>QWidget</extends>
   <header>mvdMultiResolutionPyramidWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
